lib/datadog/core/configuration/components.rb

```diff
@@ -19,11 +19,13 @@ module Configuration
       class Components
         class << self
           def build_health_metrics(settings)
-            settings = settings.diagnostics.health_metrics
-            options = { enabled: settings.enabled }
-            options[:statsd] = settings.statsd unless settings.statsd.nil?
+            health_settings = settings.diagnostics.health_metrics
 
-            Core::Diagnostics::Health::Metrics.new(**options)
+            Core::Diagnostics::Health::Metrics.new(
+              service: settings.service,
+              enabled: health_settings.enabled,
+              statsd: health_settings.statsd
+            )
           end
```

Comment:
This PR baffles me in a way. That is, this feature is SO obvious, that I cannot think why we didn't have it from the get-go. Am I missing something? How would this be usable without a `service` tag, unless customers had exactly 1 service reporting to their account?